tools/xplatRegenByteCode.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python
+#-------------------------------------------------------------------------------------------------------
+# Copyright (C) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+#-------------------------------------------------------------------------------------------------------
+
+# Regenerate embedded bytecode headers.
+# NOTEs:
+# 1. this script is for linux and macOS only, on windows please use RegenAllByteCode.cmd AND update_bytecode_version.ps1
+# 2. this script uses paths relative to the tools directory it's in so cd to it before running
+# 3. this script relies on forcing 64bit CC builds to produce 32bit bytecode - this could break due to future changes to CC.
+#    If this facility breaks the CI will fail AND either this will need fixing or bytecode will need to be regenerated using
+#    32 bit builds on windows
+# 4. Run with flag '--skip-build' if the necessary versions of ChakraCore are already built to skip the compilation step
+#    (this is useful if editing the .js files that the bytecode is produced from)
+# Two versions of CC and ch must be compiled for bytecode generation 
+# ch is used to generate bytecode.
+# ch (NoJIT variety) is used to generate NoJIT bytecodes.
+
+import subprocess, sys, uuid
+
+# Compile ChakraCore both noJit and Jit variants
+def runSub(message, commands, error):
```

Comment:
Great job on the python!

We should follow standard [PEP8](https://www.python.org/dev/peps/pep-0008/) styles for Python code, where functions and variable names are `snake_case`.